admin_manual/release_notes.rst

```diff
@@ -17,6 +18,110 @@ Release Notes
+
+SWIFT Objectstore deprecation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As the markets are moving in the direction of the S3 protocol to communicate with object storages, ownCloud will follow this path with a clear focus. For this it will be a necessity to deprecate object storage via the OpenStack SWIFT protocol. The extension will still be available as part of ownCloud Server but it will not be maintained or developed any further by ownCloud and support will be discontinued. Please make sure to move to the S3 protocol to use object storage as primary storage with future ownCloud Server versions.
+
+New options to display Imprint and Privacy Policy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To enable GDPR and legal compliance in various jurisdictions for ownCloud providers it is now possible to specify links to Imprint and Privacy Policy in the administration settings 'General' section or via the OCC commands ``php occ config:app:set core legal.imprint_url <link>`` and ``php occ config:app:set core legal.privacy_policy_url <link>``. These links can be displayed on all pages of the ownCloud web interface and in the footer of mail notifications. When using one of the default themes provided by ownCloud as well as the default mail templates, configured links will be automatically included. For customized themes or mail templates actions are required to include the links:
+
+- Add the following at the end of each HTML template to add the footer 
+
+``<?php print_unescaped($this->inc('html.mail.footer', ['app' => 'core'])); ?>``
+
+- Add the following at the end of each plain text template to add the footer 
+
+``<?php print_unescaped($this->inc('plain.mail.footer', ['app' => 'core'])); ?>``
+
+- In a custom theme change ``getShortFooter`` and ``getLongFooter`` in *defaults.php* `without links <https://github.com/owncloud/theme-example/blob/master/defaults.php#L124>`_  to `include the links <https://github.com/owncloud/core/blob/master/lib/private/legacy/defaults.php#L256>`_
```

Comment:
@pmaier1 should we mention the `getL10N()` method addition as well ?

recently @VicDeo discovered that we need that too for translations, see the linked file